docs/pages/index.mdx

```diff
@@ -8,11 +8,11 @@ videoBanner: fSgCnQrNu78
 
 Teleport is an identity-native infrastructure access platform. With Teleport you can:
 
-- Replace insecure secrets with true identity.
-- Have a single access solution for SSH, Kubernetes, databases, desktops, web applications, and more.
-- Define RBAC for every component of your infrastructure.
-- Gather audit events and session recordings to understand how users are interacting with your infrastructure.
-- Automatically onboard and offboard users with integrations such as Okta, GitHub, and Google Workspace.
+- Replace a mix of vaults, passwords, API keys and tokens with short-lived SSH and X.509 certs.
+- Have a single access point and Single-Sign-On for all your infrastructure - SSH, Kubernetes, databases, desktops, web applications, and more.
+- Write policies with Terraform or K8s CRDs for all clouds, environments and protocols and apply them with PR flow.
+- Record SSH and `kube exec` sessions, DB queries, Desktop recordings, web services and apps.
```

Comment:
- I would say `kubectl exec` if you're going to put it in backtick. 
- We say "record desktop recordings" which feels a bit repetitive.

Reply:
done